src/components/formControls/select/index.tsx

```diff
@@ -93,6 +106,72 @@ export default class Select extends React.PureComponent<Props, State> {
     }
   }
 
+  getRadioOptions = (value: string | undefined, children: React.ReactNode) => {
+    const currentValue = this.state.value
+    let options: RadioOptionsObj = {}
+
+    React.Children.map(children, (child: any, i: number) => {
+      if (child.props['data-value'] === undefined) {
+        return null
+      }
+
+      const value = child.props['data-value']
+      const selected = value === currentValue
+
+      return options[value] = selected
+    })
+
+    return options
+  }
+
+  onRadioOptionClick = (key: string, selected: boolean, child: React.ReactNode, all: {[key: string]: boolean}) => {
+    this.setState({
+      value: key,
+      selected: child,
+      radioShown: false
+    })
+  }
+
+  onOverlayClick = (event: any) => {
+    if (event.target.className.includes('StyledOptionsOverlay')) {
```

Comment:
@ryanml Is this a styled-components class name? Will this string still be present in production build mode? Is there a better way to ensure this click handler is only fired by that target? Or perhaps add and read a data-* attribute to the element

Reply:
@petemill good catch, I'll adjust this to use a data-* attribute 👍

Reply:
@petemill done